pkg/apis/agones/v1/gameserver.go

```diff
@@ -666,6 +693,12 @@ func (gs *GameServer) CountPorts(f func(policy PortPolicy) bool) int {
 func (gs *GameServer) Patch(delta *GameServer) ([]byte, error) {
 	var result []byte
 
+	if !runtime.FeatureEnabled(runtime.FeatureContainerPortAllocation) {
```

Comment:
I think this might be the wrong spot for this - I think this should be in the sections I've highlighted above.

Reply:
I wasn't sure where to actually use the feature flag - thanks for the suggestions!